hugo/layouts/papers/single.html

```diff
@@ -357,12 +377,22 @@ <h5 class="modal-title">Correct Metadata for <span id="paperIdSpan"></span></h5>
               <small id="abstractTitleHelp" class="form-text text-muted">Correct abstract if needed. Retain XML formatting tags such as &lt;tex-math&gt;.</small>
               <textarea class="form-control" id="paperAbstract" rows="6"></textarea>
             </div>
+
+            <div class="mb-3">
+              <label class="form-label">Verification against PDF</label>
+              <small class="form-text text-muted">Ensure that the new title/authors match the snapshot below. (If there is no snapshot or it is too small, consult <a href="#" id="paperPDF">the PDF</a>.)</small>
+              <div style="height: 150px; text-align: center; overflow: hidden; position: relative;">
+                <a id="paperSnapshot" href=""><img id="paperSnapshotImg" src="" style="position: absolute; left: 0; top: -50px;"></a>
+              </div>
```

Comment:
I played around with this a bit and think something like this might work:

```html
<div style="max-height:150px;" class="overflow-hidden w-100">
  <a id="paperSnapshot" href="https://aclanthology.org/thumb/2024.tacl-1.3.jpg">
    <img id="paperSnapshotImg" src="https://aclanthology.org/thumb/2024.tacl-1.3.jpg" class="w-100" style="transform: translate(0, 5%) scale(1.25);">
  </a>
</div>
```

Reply:
I played quite a bit but eventually had too many hours into the project and was happy to have something working. This kind of design isn't my forte. I'll try pushing up this change on this PR.